src/index.html

```diff
@@ -11,7 +11,152 @@
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+             type="number"
+             name="age"
+             min="1"
+             max="100"
+             value="12"
+             required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+             type="date"
+             name="dateOfbirth"
+             required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agree"
+            required
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="25"
+            required
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <input type="radio" name="cats">Yes
+          <input type="radio" name="cats">No
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            value="black"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            step="1"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select multiple name="cars">
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            value="0"
+            required
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+         ></textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+
+      </fieldset>
```

Comment:
```suggestion
        </label>
      </fieldset>
```
There is no need to add an empty line between the child and the parent.